### ISSUE TITLE
Pressing "Ignore" should disable the validation

### ISSUE THREAD

**Issue by wwalc** [cksource/ckeditor-plugin-a11ychecker#67]
Using the default sample, I moved to the problem with missing alt text:

![image](https://cloud.githubusercontent.com/assets/545175/4917928/5525069e-64eb-11e4-8cf0-0f0f9e481c8f.png)

When I pressed "Ignore" button this alert was shown:

![image](https://cloud.githubusercontent.com/assets/545175/4917911/2232435a-64eb-11e4-8e36-c522d2747fb5.png)
